### PR TITLE
Updated NTP IP list, added .vs directory in .gitignore

### PR DESCRIPTION
**Pull request by gtrevi**
# Description

Updated NTP IP list, added .vs directory in .gitignore

## Type of change

Please delete options that are not relevant.

- Update to existing content

# Checklist:

- [X ] My content has a file named README.md, which is based on the appropriate readme [template](https://github.com/Azure/azure-sphere-gallery/tree/main/Templates)
- [X ] I have performed a self-review of my own contribution
- [X ] I have provided comments where appropriate
- [X ] I have updated the repository's [README.md](https://github.com/Azure/azure-sphere-gallery/blob/main/README.md) to index this project
- [X ] I have added/updated license(s) for this project as appropriate
- [X ] I have permission to publish this content (in case the content was collaborative)
- [X ] I have verified that my content does not contain sensitive information (PII, Microsoft PI, etc.) and is safe to open source

### CODE REVIEW

**Review by miz060 — APPROVED** [review]

**Review by kenwoodberry — APPROVED** [review]

File: OSNetworkRequirementChecker-HLApp/src/ntp-helper.c [hunk 1]
```diff
@@ -8,10 +8,10 @@
 #define NTP_SERVER_LEN 256
 
 // List of time server to be tested
-const char *NTPServerList[] = {"13.86.101.172",  "20.43.94.199",   "20.189.79.72",
-                               "40.81.94.65",    "40.81.188.85",   "40.119.6.228",
-                               "51.105.208.173", "51.137.137.111", "51.145.123.29",
-                               "52.148.114.188", "52.231.114.183"};
+const char *NTPServerList[] = { "168.61.215.74", "20.43.94.199", "20.189.79.72",
+                                "40.81.94.65", "40.81.188.85", "40.119.6.228", "40.119.148.38",
```
Comment (cawhitworth) [inline OSNetworkRequirementChecker-HLApp/src/ntp-helper.c:12]: Does the README need a comment, as for the PC version, that 40.81.188.85 may fail if using a commercial ISP in the US?

Comment (gtrevi) [reply OSNetworkRequirementChecker-HLApp/src/ntp-helper.c:12]: Yes, totally agree, @miz060 I pushed the update in the README.md, if that's ok with you. Thanks.

Comment (cawhitworth) [reply OSNetworkRequirementChecker-HLApp/src/ntp-helper.c:12]: That looks reasonable to me.

**Review by cawhitworth — APPROVED** [review]

File: OSNetworkRequirementChecker-HLApp/src/ntp-helper.c (hunk 1 above)
Comment (miz060) [reply OSNetworkRequirementChecker-HLApp/src/ntp-helper.c:12]: @gtrevi Yes, that is the case. With the SNTP client fallback mechanism in place, the device should always be able to complete time sync eventually (except when all time server IPs are not reachable).

Comment (gtrevi) [reply OSNetworkRequirementChecker-HLApp/src/ntp-helper.c:12]: @miz060 Perfect, totally agree and just pushed the mods to disable the fallback (+ consequent README.md update).

Comment (miz060) [reply OSNetworkRequirementChecker-HLApp/src/ntp-helper.c:12]: @gtrevi thanks! the update looks good to me.

**Pull request merged**